cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -2,7 +2,10 @@
 Distributed training using Horovod on Spark
```

Comment:
```suggestion
Data-Parallel Distributed Training Using Horovod on Spark
```
Can we also have some intro here? One or two lines should suffice.

Reply:
Sure, I will add an introduction